Docs/Configuration.tex

```diff
@@ -6828,6 +6828,51 @@ \subsection{AppleInput Properties}\label{uefiappleinputprops}
   \item \texttt{false} --- Prevent key input mirroring to non-Apple protocols when in graphics mode.
   \end{itemize}
 
+  \item
+  \texttt{PointerPollMin}\\
+  \textbf{Type}: \texttt{plist\ integer}\\
+  \textbf{Failsafe}: \texttt{0}\\
+  \textbf{Description}: Configure minimal pointer polling period in ms.
+
+  This is the minimal period AppleEvent polls pointer devices (e.g. mice, trackpads)
+  for motion events. Defaults to 10 ms in the OpenCore AppleEvent implementation
+  and to 2 ms in the OEM Apple implementation.
```

Comment:
Should probably explicitly state "when 0" because we have no defaults. :) But hmm, for TiimerResolution "0" means "unchanged", and "0" here just picks a hardcoded default, why not just set the default in the Sample as with more properties?

Reply:
The issue is the OEM and Builtin implementations, which have different defaults. I do not want this to be confusing.